connectors/src/connectors/google_drive/lib.ts

```diff
@@ -7,23 +7,100 @@ import {
   Ok,
 } from "@dust-tt/types";
 import type { InferAttributes, WhereOptions } from "sequelize";
+import { Op } from "sequelize";
 import { v4 as uuidv4 } from "uuid";
 
+import { GOOGLE_DRIVE_WEBHOOK_LIFE_MS } from "@connectors/connectors/google_drive/lib/config";
+import { getGoogleDriveObject } from "@connectors/connectors/google_drive/lib/google_drive_api";
+import {
+  getDrivesIdsToSync,
+  getSyncPageToken,
+} from "@connectors/connectors/google_drive/temporal/activities";
 import { isGoogleDriveSpreadSheetFile } from "@connectors/connectors/google_drive/temporal/mime_types";
-import { getAuthObject } from "@connectors/connectors/google_drive/temporal/utils";
+import {
+  getAuthObject,
+  getDriveClient,
+} from "@connectors/connectors/google_drive/temporal/utils";
 import { HTTPError } from "@connectors/lib/error";
 import {
   GoogleDriveFiles,
   GoogleDriveSheet,
+  GoogleDriveWebhook,
 } from "@connectors/lib/models/google_drive";
+import logger from "@connectors/logger/logger";
 import type { ConnectorResource } from "@connectors/resources/connector_resource";
 import type { ConnectorModel } from "@connectors/resources/storage/models/connector_model";
 
 const { CONNECTORS_PUBLIC_URL, DUST_CONNECTORS_WEBHOOKS_SECRET } = process.env;
 
+export async function registerWebhooksForAllDrives(
```

Comment:
NIT: `registerWebhooksForDrivesToSync`? (not sure relevant, sugggestion to be consistent with getDrivesIdsToSync )